CHANGES.md

```diff
@@ -1,5 +1,15 @@
 # Change log for kotlinx.coroutines
 
+## Version 1.6.2
+
+* Fixed a bug with `ThreadLocalElement` not being correctly updated when the most outer `suspend` function was started directly without `kotlinx.coroutines` (#2930).
+* Fixed multiple data races that might have been affecting `Mutex` and `runBlocking` event loop (#3250, #3251).
```

Comment:
The data race in `Mutex` was benign (that is, shouldn't affect anything in practice), maybe we should highlight it.

Reply:
I'm not sure it really matters here

Reply:
I just think that reading the changelog and seeing that `Mutex` had a data race is pretty scary, given that its whole purpose is to guard against them.